src/components/userComponents/Exhibit/Exhibit.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Image from 'next/image';
+
+/**
+ *
+ * @param root0 passed in
+ * @param root0.title title of exhibit
+ * @param root0.description description of exhibit
+ * @param root0.category category of exhibit
+ * @param root0.image image
+ * @param root0.id id of exhibit
+ * @returns exhibit component
+ */
+export default function Exhibit({
```

Comment:
this is an example of something that could be refactored to match both the web and mobile screen versions, but this is something we can talk about later @sarahhpeng @jxmoose